arch/arm/boot/dts/overlays/revpi-flat-s-2022-overlay.dts

```diff
@@ -0,0 +1,378 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Flat S (2022)
+ */
+
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/{
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-flat-s-2022",
+				     "kunbus,revpi-flat", "brcm,bcm2711";
+
+			aout_vref: fixedregulator_2v5x4 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x4_amplified";
+				regulator-min-microvolt = <10000000>;
+				regulator-max-microvolt = <10000000>;
+				regulator-always-on;
+			};
+
+			ain_vref: fixedregulator_2v5x5 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x5_amplified";
+				regulator-min-microvolt = <12500000>;
+				regulator-max-microvolt = <12500000>;
+				regulator-always-on;
+			};
+
+			wifi_pwrseq: wifi_pwrserq {
+				compatible = "mmc-pwrseq-simple";
+				reset-gpios = <&expander 12 GPIO_ACTIVE_LOW>;
+			};
+
+			leds {
+				compatible = "gpio-leds";
```

Comment:
Here we need a pinmux for gpio 24. The power LED

Reply:
Ok

Reply:
After studying the schematics I still do not see why this is needed. Could you shed some light on this?

Reply:
We are using GPIO24 for the power led. So we should configure the pin with the correct function. The other leds are connected to the gpio expander. The expander can't have any other function then GPIO so we don't need to do this fore these pins.

Reply:
This is not about any pulls it's just about the function. It will work as it is, but this is only because the default function of most pins is GPIO. And if a gpio is requested, the gpio controller will just change its function. But this is more a bug then a proper behavior of the gpio/pin controller

Reply:
Ok got it, thx for the explanation.